src/MatdaAIga.LinkConverter/Options/ArgumentOptions.cs

```diff
@@ -0,0 +1,54 @@
+namespace MatdaAIga.LinkConverter.Options;
+
+/// <summary>
+///  Provides the options for the command line arguments
+/// </summary>
+public class ArgumentOptions
+{
+    /// <summary>
+    /// Gets or sets a value indicating whether to display help or not.
+    /// </summary>
+    public bool Help { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets the filepath of the data source.
+    /// </summary>
+    public string Filepath { get; set; } = string.Empty;
+
+    /// <summary>
+    /// Parses the command line arguments and returns the options.
```

Comment:
`/// Parses the command line arguments to <see cref="ArgumentOptions" /> object`